src/UIComponents/Button/Button.js

```diff
@@ -1,50 +1,51 @@
 import React from 'react'
+import { Link } from 'react-router-dom'
 import PropTypes from 'prop-types'
+import './Button.css'
 
-const Button = (props) => {
-  const { children, type, disabled } = props
-  // this allows us to pass any text as child
-  // create different types of buttons
-  // also we can pass in a boolean prop into disabled so if it is loading, it is disabled
+const Button = ({
+  children,
+  href,
+  to,
+  inverse,
+  style,
+  disabled,
+  type,
+  onClick,
+}) => {
```

Comment:
Awesome addition for additional properties. This will make the button much more versatile.